pkg/operator/controller/gatewayclass/subscription.go

```diff
@@ -53,11 +53,14 @@ func desiredSubscription(name types.NamespacedName) (*operatorsv1alpha1.Subscrip
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: name.Namespace,
 			Name:      name.Name,
+			Annotations: map[string]string{
+				operatorcontroller.IngressOperatorOwnedAnnotation: "",
+			},
```

Comment:
Do we have to add this new annotation to function `routerNamespaceChanged` in pkg/operator/controller/ingress/namespace.go?